INSTRUCTION.md

```diff
@@ -0,0 +1,17 @@
+how to run MySQL container with a volume attached
+docker run -d -p 3306:3306 --name mysql-local \
+-e MYSQL_ROOT_PASSWORD=1234 \
```

Comment:
Using hardcoded passwords like `MYSQL_ROOT_PASSWORD=1234` is a security risk. Consider using environment variables or a more secure method to handle sensitive information.